dublast/panels.py

```diff
@@ -0,0 +1,208 @@
+import bpy
+from bpy.types import Panel
+
+class DUBLAST_PT_Playblast_Settings( Panel ):
+    bl_label = "Playblast"
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "output"
+    bl_options = {'DEFAULT_CLOSED'}
+    COMPAT_ENGINES = {'BLENDER_RENDER'}
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        layout.use_property_decorate = False  # No animation.
+        playblast_settings = bpy.context.scene.playblast
+        rows = layout.row(align=False)
+        rows.operator('render.playblast', text="Playblast", icon= 'FILE_MOVIE')
+        row = rows.row(align=True)
+        row.prop(playblast_settings, 'increment', icon="LINENUMBERS_ON", toggle=True)
+        row.prop(playblast_settings, 'use_stamp', icon="INFO", toggle=True)
+        row.prop(playblast_settings, 'include_annotations', icon="TEXT", toggle=True)
+
+class DUBLAST_PT_Scene( Panel ):
+    bl_label = ""
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "output"
+    bl_options = {'DEFAULT_CLOSED'}
+    bl_parent_id = "DUBLAST_PT_Playblast_Settings"
+
+    def draw_header(self,context):
+        self.layout.label(text="Scene", icon="SCENE_DATA")
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = False
+        layout.use_property_decorate = False  # No animation.
+        # Add settings for the current scene
+        playblast_settings = bpy.context.scene.playblast
+        
+        col = layout.column(align=False)
+        row = col.row(align=True)
+        #row.label(text='Scene')
+        row.prop( playblast_settings, "use_camera", icon="VIEW_CAMERA", toggle=True)
+        row.prop( playblast_settings, "use_scene_frame_range", icon="PREVIEW_RANGE", toggle=True )
```

Comment:
These should be regular checkboxes on a column, as they're independent settings; as toggles on a row it seems they're connected in some way, and may take too much space in small UI
![image](https://user-images.githubusercontent.com/8526542/174755296-df126bb5-b5b3-4db8-8fad-abfe333bea00.png)